author/ogc/authoring-guide/migrating-docs.adoc

```diff
@@ -0,0 +1,388 @@
+AsciiDoc documents to Metanorma syntax.
+
+== OGC attributes
+
+The only document attributes to use must be those officially supported
+by Metanorma:
+https://www.metanorma.org/author/ref/document-attributes/[_generic-purpose attributes_]
+or https://www.metanorma.org/author/ogc/ref/document-attributes/[_flavor-specific attributes_].
+
+Any other attribute not present in the official documentation should
+be dismissed.
+
+== Tables
+
+Tables in Metanorma must be encoded using the basic Asciidoctor approach:
+a table delimiter be encoded as `|===`, and a column separator `|`.
+
+For example:
+
+[source,asciidoc]
+----
+[cols="1,1"]
+|===
+|Cell in column 1, row 1
+|Cell in column 2, row 1
+
+|Cell in column 1, row 2
+|Cell in column 2, row 2
+
+|Cell in column 1, row 3
+|Cell in column 2, row 3
+|===
+----
+
+No other character for table delimiter or column separator should
+be used.
+
+When it comes to composing complex tables, Metanorma permits a limited
+number of attributes given in https://www.metanorma.org/author/topics/document-format/text/#tables.
+
+NOTE: Even when `width` attribute (for establishing fixed table width)
+is supported, its use is not recommended as it changes the preset
+formatting of the flavor.
+
+== Cross-references
+
+In most cases, we shouldn't use text substitution
+(e.g. `\<<anchor-id,custom text>>`) when we are cross referencing to
+internal elements of the document, like: clauses, tables, images,
+notes, examples, etc.
+
+Using just `\<<anchor-id>>` will lead to the proper identification
+of the cross-referenced element.
+
+=== More information
+
+General use of cross-references:
+https://www.metanorma.org/author/topics/document-format/xrefs/#text-cross-refs +
+Anchor ID syntax:
+https://www.metanorma.org/author/topics/document-format/xrefs/#text-ref-allowed-anchors
+
+== References
+
+References must be encoded as indicated in
+https://www.metanorma.org/author/topics/document-format/bibliography/[official documentation].
+In OGC flavor, the section of _Normative references_ will generate
+a boilerplate paragraph at the beginning of the section that states:
+
+____
+The following documents are referred to in the text in such a way
+that some or all of their content constitutes requirements of this
+document. For dated references, only the edition cited applies. For
+undated references, the latest edition of the referenced document
+(including any amendments) applies
+____
+
+We can override this text using a NOTE with `boilerplate` attribute:
+
+[source,asciidoc]
+----
+[NOTE,type=boilerplate]
+--
+<custom text>
+--
+----
+
+However, it should be remembered that predefined texts are supposed
+to be compliant with the flavor authoring specifications, so the overriding
+is generally not recommended.
+
+In relation to the encoding of bibliographic references,
+some examples are given below to illustrate the options available:
+
+.Typical reference entry (without auto-fetch)
+====
+[source,asciidoc]
```

Comment:
We recommend in the order of:
1. Auto-fetch from document identifier
2. Auto-fetch from DOI or OpenLibrary
3. Citation span encoding
4. Plain text.

Reply:
@ronaldtse I've changed the order but, the related content is not a recommendation. We just tried to showcase the options available to encode bibliographic references. For instance, references with *auto-fetch, auto-fetch disabled, plain text, user-supplied label, and numeric label. *

Reply:
OK